doc/developer/topotests.rst

```diff
@@ -360,6 +360,7 @@ This is the recommended test writing routine:
 - Write a topology (Graphviz recommended)
 - Obtain configuration files
 - Write the test itself
+- Format the new code using `black <https://github.com/psf/black>`__
```

Comment:
shouldn't this be a single underscore? ditto 764

Reply:
looks legit:

![Screenshot from 2020-04-05 18-39-03](https://user-images.githubusercontent.com/3352707/78503020-c1cf8f00-776c-11ea-9eba-4070572467f0.png)

or what did you mean? :)

Reply:
from e.g. `bgp.rst`:
```
BGP graceful restart functionality as defined in                                    
`RFC-4724 <https://tools.ietf.org/html/rfc4724/>`_ defines the mechanisms that     
allows BGP speaker to continue to forward data packets along known routes             
while the routing protocol information is being restored. 
```

Reply:
:+1:

Reply:
![Screenshot from 2020-04-05 20-18-00](https://user-images.githubusercontent.com/3352707/78505219-a3709000-777a-11ea-9bc6-a53a98bd8ae1.png)

> if in doubt, use two :rofl: